shared/templates/docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Galaxy - <topic name> training material
+
+FROM bgruening/<base-galaxy-flavor>
```

Comment:
This does not need to be under my namespace, actually it would be good to have them at some point in an organisation. Who knows when a bus hits me.